src/packages.jl

```diff
@@ -188,6 +194,7 @@ end
 """
     install(spec::String, version::String = "";
                           interactive::Bool = true, quiet::Bool = false,
+                          debug::Bool = false,
```

Comment:
Of course this should also be documented; I'll do that once we settled on what final form this should have.